net-im/telegram-desktop-bin/telegram-desktop-bin-1.6.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils xdg
```

Comment:
gnome2-utils isn't needed for icon_cache_update anymore, it's included in xdg-utils. I see you inherit xdg and call for the default phase setups, if you update EAPI to 7, those should now also include xdg_icon_cache_update.